iele-gas.md

```diff
@@ -1012,16 +1066,19 @@ This schedule is used to execute the EVM VM tests, and contains minor variations
 
     rule Gselfdestructnewaccount << DEFAULT >> => false
     rule Gstaticcalldepth        << DEFAULT >> => true
+    rule Gnewmove                << DEFAULT >> => false
+    rule Gnewarith               << DEFAULT >> => false
 
     rule Smemallowance  < DEFAULT > => 4096
+    rule Sgasdivisor    < DEFAULT > => 1
 ```
 
 ### Albe Schedule
 
 This is the initial schedule of IELE.
 
 ```k
-    syntax Schedule ::= "ALBE"
+ // Albe
```

Comment:
Where did `syntax Schedule ::= "ALBE"` go?

Reply:
It was moved to well-formedness.k because it's needed now by the well-formedness semantics.